content/docs/result-components/ReactiveList.md

```diff
@@ -182,6 +182,16 @@ Example uses:
     show custom message or component when no results found.
 - **renderPagination** `Function` [optional]
     can be used to render custom pagination.
+     - **`pages`**: `number`
+        Number of pages to be displayed
+    - **`totalPages`**: `number`
+        Total number of pages found based on current page size
+    - **`currentPage`**: `number`
+        Current page number for which data is being rendered
+    - **`setPage`**: `function`
+        Function use to set the currentPage
+    - **`fragmentName`**: `string`
+        `componentId` of the Result component for setting url params
```

Comment:
@jyash97 How that can be used by a user?

Reply:
Using ```<a href={`?${fragmentName}={pageNumber}`} />```

This how we do in our Pagination component

Reply:
It is helpful, can you add it too in docs.

Reply:
Done